add-chain/main.go

```diff
@@ -90,7 +90,6 @@ func entrypoint(ctx *cli.Context) error {
 	sequencerRPC := ctx.String(flags.SequencerRpcFlag.Name)
 	explorer := ctx.String(flags.ExplorerFlag.Name)
 	superchainTarget := ctx.String(flags.SuperchainTargetFlag.Name)
-	monorepoDir := ctx.String(flags.MonorepoDirFlag.Name)
```

Comment:
Were the changes to this file intentional?

Reply:
Good catch. They are intentional but I will separate those changes into a different pr to keep this one cleaner